### ISSUE TITLE
chromium: google translate not working (suspect API key issue)

### ISSUE THREAD

**Issue by ackalker** [voidlinux/void-packages#5672]
When I visit a page in a foreign language, a popup offers to translate the page, but when I confirm I wish to have it translated and the popup changes to "This page has been translated", text remains in the original language.

Steps to reproduce:

* Visit a page in foreighn language, for example: https://www.art-system.ru/
* Choose "Translate this page" in the popup.
* Wait for confirmation "This page has been translated".

The JavaScript console (open with Ctrl-Shift-j) shows some interesting messages:
```
VM724:202 POST https://translate.googleapis.com/translate_a/<redacted> 403 ()
[..snip..]
VM724:202 POST https://translate.googleapis.com/translate_a/<redacted> 403 ()
[..snip..]
```

Status code 403 is the classical "Forbidden" status, but from some searching, I get the notion that Google Translate may be blocking the request because of problems with the API key sent by Chromium for Void. The API key is sent by Chromium, I could confirm this by looking at the request headers in Developer Tools.

AFAICT, the API key and other data was set by commit 160f39c38b85914fd706cda318aa6c820a9a36b9.

**Comment by ackalker** [comment]
Another possibility I found from some searching: since these API keys are meant to be used by a *single user*, not redistributed, it could be that we're hitting some kind of quota limit, since all Void users using Chromium are using the same API keys. See https://www.chromium.org/developers/how-tos/api-keys for more information.

**Comment by ackalker** [comment]
I see that Debian is using API keys which they claim were assigned by Google: https://anonscm.debian.org/cgit/pkg-chromium/pkg-chromium.git/tree/debian/apikeys?id=refs/heads/master

**Comment by chneukirchen** [comment]
Can confirm overrinding Debian's API key instead of ours works... so why did our key get canceled?

**Comment by ackalker** [comment]
The commit message says that the keys were provided by @mrneilypops, perhaps he can provide some insight.
Could it be that the project on https://cloud.google.com/console which uses these keys was cancelled? Has some key expired perhaps?

**Comment by chneukirchen** [comment]
Fixed in 0a720cdad8e.

**Comment by TheGrave** [comment]
Last couple of version issue reappears (I'm on 73 now, Ubuntu 18.04). Could it be the same story? Chrome works just fine so I guess it's not broken code.
